src/applications/disability-benefits/all-claims/utils/validations.js

```diff
@@ -1,241 +1,110 @@
-import moment from 'moment';
 import { convertToDateField } from '~/platform/forms-system/src/js/validation';
 import { isValidDateRange } from '~/platform/forms/validations';
 
-/* The particular field used for the Toxic Exposure date inputs properly executes
-* validateCurrentOrPastDate, however schema validations against the individual fields
-* (day, month, year) do not properly block the Veteran from progressing.
-* These validations are necessary to properly block progression and prevent what
-* we're calling the 'XX' date issue.
-*/
-export function validateToxicExposureGulfWar1990Dates(
-  errors,
-  { startDate, endDate },
-) {
-  // With these fields in particular, a missing value will convert to
-  // a date format with 'XX' e.g. 01-XX-1990.
-  // Calling convertToDateField on a partial date will return an empty date object.
-  // We will validate around this fact.
+const messages = {
+  invalidRange: 'Enter a service start date that’s before your end date.',
+  endDate1990: 'Enter a service end date after August 2, 1990.',
+  endDate2001: 'Enter a service end date after September 11, 2001.',
+  // This is largely redundant because the component validation already prevents
+  // inputting dates in the future. However, this is in place just in case, because...
+  // TODO: The current schema validation says that the Veteran can input a year
+  // up to 100 years in the future. That is incorrect and should be corrected/removed.
+  futureDate: 'Enter a service year that’s before or during the current year.',
```

Comment:
I assume this content was approved, but I wonder if instructing the user to modify the year might be misleading, when the problem might instead be a month or day value that is in the future.

Reply:
This message content does not appear in the linked Figma Designs.